src/components/Button/index.js

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { cx } from "emotion"
+import {
+  buttonCls,
+  primaryBtn,
+  secondaryBtn,
+  dangerBtn,
+  ghostBtn
+} from "./style.js"
+
+const ButtonPropTypes = {
+  className: PropTypes.string,
+  type: PropTypes.string,
+  onClick: PropTypes.func,
+  disable: PropTypes.bool,
+  variant: PropTypes.oneOf(["primary", "secondary", "danger"]),
+  size: PropTypes.oneOf(["small", "large"]),
+  ghost: PropTypes.bool
+}
+
+function SubmitButton({ className, onClick = () => {}, size, variant }) {
+  return (
+    <button
+      onClick={(e) => {
+        e.preventDefault()
+        onClick(e.target.value)
+      }}
```

Comment:
I think that this would be sufficient
```suggestion
      onClick={onClick}
```